larray/core/array.py

```diff
@@ -1473,14 +1473,175 @@ def get_labels(self_axis):
         else:
             return res
 
+    def align(self, other, join='outer', fill_value=nan, axes=None):
```

Comment:
Need also a comment about anonymous axes (maybe a doctest that raises an Error).